apps/mobile/src/App.tsx

```diff
@@ -35,6 +36,7 @@ function App(): JSX.Element {
     <SafeAreaProvider>
       <StatusBar style="light" />
       <NavigationContainer
+        ref={navigationRef}
```

Comment:
This is the way how to dispatch navigation events outside of the react tree: [official documentation](https://reactnavigation.org/docs/navigating-without-navigation-prop/)